wgsl/index.bs

```diff
@@ -5609,19 +5618,13 @@ The following statement forms are equivalent:
 A <dfn noexport dfn-for="statement">break</dfn> statement transfers control to the first statement
 after the body of the nearest-enclosing [=statement/loop=]
 or [=statement/switch=] statement.
-A `break` statement must only be used within [=statement/loop=], [=statement/for=], and [=statement/switch=] statements.
 
-When a `break` statement is placed such that it would exit from a loop's [=statement/continuing=] statement,
-then:
+A `break` statement must only be used within [=statement/loop=], [=statement/for=], and [=statement/switch=] statements.
 
-* The `break` statement must appear as either:
-    * The only statement in the `if` clause of an `if` statement that has:
-        * no `else` clause or an empty `else` clause
-        * no `else if` clauses
-    * The only statement in the `else` clause of an `if` statement that has an empty `if` clause and no `else if` clauses.
-* That `if` statement must appear last in the `continuing` clause.
+A `break` statement must not placed such that it would exit from a loop's [[#continuing-statement|continuing]] statement.
```

Comment:
This seems a little roundabout.
```suggestion
A `break` statement must not used in a [[#continuing-statement|continuing]] statement.
```

Reply:
The distinguishing case is a loop nested in a loop.

```
fn main() {
   loop {
      continuing {
             loop {
                break;  // this is fine. It escapes the inner loop, not the continuing block of the outer loop.
             }
      }
   }
}
```
I don't want anyone to argue that that break is invalid.